packages/react-native/React/Base/RCTUtils.h

```diff
@@ -95,6 +95,9 @@ RCT_EXTERN UIApplication *__nullable RCTSharedApplication(void);
 // or view controller
 RCT_EXTERN UIWindow *__nullable RCTKeyWindow(void);
 
+// Returns the current window interface orientation
+RCT_EXTERN UIInterfaceOrientation RCTUIInterfaceOrientation(void);
+
```

Comment:
Can we delete this and unwrap the code directly in the calling site?
The reason is that this will be an extra public API that we will have to maintain and if we decide to change something it can generate breaking changes.
The `statusBarOrientation` didn't have a public API that anyone could read, so it's better to keep it that way. We can always add it back later if we need it.

The tradeoff is a very little code duplication for a lower maintenance cost.

Reply:
i understand :-)